.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+name: Sync master to main
+on:
+  push:
+    branches:
+      - "master"
+jobs:
+  diff:
+    runs-on: ubuntu-latest
+    outputs:
+      hash-master: ${{ steps.hash-master.outputs.hash-master }}
+      hash-main: ${{ steps.hash-main.outputs.hash-main }}
+    steps:
+    - uses: actions/checkout@v3
```

Comment:
What do you think to use `actions/checkout@v4`?

Reply:
Good catch. Thanks!